core/src/js/components/battlegrounds/graph-with-comparison-new.component.ts

```diff
@@ -57,6 +57,9 @@ export class GraphWithComparisonNewComponent {
 	@Input() yourLabel = 'You';
 	@Input() communityTooltip: string;
 	@Input() yourTooltip: string;
+	@Input() turnLabel = 'Turn';
+	@Input() statLabel = 'Stat';
```

Comment:
Shouldn't localized strings be used here as well?

Reply:
I left it as some default values just in case the values don't get passed. It seems to me that displaying something is better than displaying nothing)
If you want to localize this too, then we need to use LocalizationFacadeService,  we are already passing too many parameters to this component.